mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.java

```diff
@@ -184,26 +203,30 @@ public void onItemLongClicked(final Widget widget) {
 
         if (widget.linkedPage() != null) {
             labels.add(getString(R.string.nfc_action_to_sitemap_page));
+            labels.add(getString(R.string.home_shortcut_pin_to_home));
```

Comment:
Should check https://developer.android.com/reference/androidx/core/content/pm/ShortcutManagerCompat.html#isRequestPinShortcutSupported(android.content.Context) before adding this.